src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, DrawingFigure {
+    private String color;
+

```

Comment:
```suggestion
    private String color;

    public Figure(String color) {
        this.color = color;
    }
```
lets use constructor